command/alloc_status.go

```diff
@@ -283,6 +283,13 @@ func formatAllocBasicInfo(alloc *api.Allocation, client *api.Client, uuidLength
 		basic = append(basic,
 			fmt.Sprintf("Replacement Alloc ID|%s", limit(alloc.NextAllocation, uuidLength)))
 	}
+	if alloc.FollowupEvalID != "" {
+		nextEvalTime := futureEvalTimePretty(alloc.FollowupEvalID, client)
+		if nextEvalTime != "" {
+			basic = append(basic,
+				fmt.Sprintf("Reschedule Eligibility|%s", nextEvalTime))
```

Comment:
Don't really like "Reschedule Eligibility". What about "Rescheduling In"?

Reply:
"Rescheduling In" won't look right when the alloc has already been rescheduled before you run alloc status. I tried to come up with a phrase that works both directions, so it would read correctly e.g "Reschedule Eligibility: 1hr from now" or "Reschedule Eligibility: 5m ago". 

Maybe we can make it just the phrase "Rescheduling"?

Reply:
Hmm okay with that context lets keep it as it is. We can try to refine it later.